8bitdo-firmware.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+# based on https://ladis.cloud/blog/posts/firmware-update-8bitdo.html
+
+import requests,json,sys,os
+import urllib.request
+
+baseurl = "http://dl.8bitdo.com:8080"
+
+things = {}
+
+def halp():
```

Comment:
help? Maybe this could use argparse instead?

Reply:
Renamed, but I don't think it's worth going for the entire argparse shebang here?